azure/Guidelines.md

```diff
@@ -145,24 +59,27 @@ Where:
  | resource&#x2011;collection | Name of the collection, unabbreviated, pluralized
  | resource&#x2011;id | Value of the unique id property. This MUST be the raw string/number/guid value with no quoting but properly escaped to fit in a URL segment.
 
-:white_check_mark: **DO** use kebab-casing (preferred) or camel-casing for URL path segments. If the segment refers to a JSON field, use camel casing.
+> :white_check_mark: **DO** use kebab-casing (preferred) or camel-casing for URL path segments. If the segment refers to a JSON field, use camel casing.
+> 
+> :white_check_mark: **DO** return '414-URI Too Long' if a URL exceeds 2083 characters
```

Comment:
We are not consistent in our treatment of response codes.  Sometimes they are "code-quoted" (back-ticks) and sometimes not.  But this is the only place where a response code is single quoted, so I'm wrapping in back-ticks to make it consistent with at least some of the other response codes.
```suggestion
> :white_check_mark: **DO** return ```414-URI Too Long``` if a URL exceeds 2083 characters
```